frontend/packages/operator-lifecycle-manager/src/const.ts

```diff
@@ -19,14 +19,16 @@ export enum DefaultCatalogSourceDisplayName {
   Custom = 'Custom',
 }
 
-export const OPERATOR_UNINSTALL_MESSAGE_ANNOTATION = 'operator.openshift.io/uninstall-message';
-export const OPERATOR_TYPE_ANNOTATION = 'operators.operatorframework.io/operator-type';
-export const NON_STANDALONE_ANNOTATION_VALUE = 'non-standalone';
-export const INTERNAL_OBJECTS_ANNOTATION = 'operators.operatorframework.io/internal-objects';
-export const DEFAULT_SOURCE_NAMESPACE = 'openshift-marketplace';
-export const OPERATOR_PLUGINS_ANNOTATION = 'console.openshift.io/plugins';
 export const DEFAULT_GLOBAL_OPERATOR_INSTALLATION_NAMESPACE = 'openshift-operators';
+export const DEFAULT_SOURCE_NAMESPACE = 'openshift-marketplace';
 export const GLOBAL_COPIED_CSV_NAMESPACE = 'openshift';
+export const INTERNAL_OBJECTS_ANNOTATION = 'operators.operatorframework.io/internal-objects';
+export const NON_STANDALONE_ANNOTATION_VALUE = 'non-standalone';
+export const OPERATOR_NAMESPACE_ANNOTATION = 'olm.operatorNamespace';
```

Comment:
nit: could we are use this const on other places in the code and remove the hard-coded string